sound/soc/sof/intel/apl.c

```diff
@@ -74,6 +74,7 @@ const struct snd_sof_dsp_ops sof_apl_ops = {
 	.pcm_hw_free	= hda_dsp_stream_hw_free,
```

Comment:
commit: "Rewinds are only used by PulseAudio"
That's a bit strong wording. There are thousands of ALSA applications out there and I'm sure rewinds are used by some of them. Maybe "Rewinds are not used by many applications. One notable application using rewinds is PulseAudio. Practical experiments..:"

Reply:
thanks @kv2019i, I used your wording verbatim, good suggestion.